src/pages/api/progress.ts

```diff
@@ -0,0 +1,85 @@
+import TinCan from 'tincanjs'
+import axios from 'axios'
+import {NextApiRequest, NextApiResponse} from 'next'
+import {connectScrollTo} from 'react-instantsearch-dom'
+
+const SCORM_CLOUD_ENDPOINT = `https://cloud.scorm.com/lrs/PZ9CYEKKV8/`
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  console.log(req.query)
+  if (req.method === 'GET') {
+    try {
+      const lrs = new TinCan.LRS({
+        endpoint: SCORM_CLOUD_ENDPOINT,
+        username: process.env.SCORM_CLOUD_USERNAME,
```

Comment:
creds are stored in 1password to log in to the scorm cloud site our in dev by using `vercel dev`